// devdot/config.js
export const author = {
  email: 'yhjor@me.com',
  name: 'George Jor',
  url: 'yhjor.me',
};

export const install = {
  apm: [
    'file-icons',
    'todo-show',
    'mocha-test-runner',
    'npm-install',
    'pigments',
    'pretty-json',
    'autocomplete-paths',
    'copy-path',
    'linter',
    'linter-eslint',
    'editorconfig',
    'es6-javascript',
    'react',
    'language-babel',
    'language-javascript-jsx',
    'atom-ternjs',
    'project-switcher',
    'line-jumper',
    'jumpy',
    'git-blame',
    'git-history',
    'git-time-machine',
    'merge-conflicts',
    'show-origin',
    'line-diff-details',
    'branch-status',
  ],
  brew: [
    'mongodb',
    'redis',
    'httpie',
    'nvm',
    'heroku-toolbelt',
    'ccat',
    'rbenv',
    'tree',
  ],
  cask: [
    'java',
    'firefox',
    'google-chrome',
    'atom',
    'virtualbox',
    'mongohub',
    'macdown',
    'skype',
    'slack',
    'evernote',
    'cakebrew',
    'spotify',
  ],
  npm: [
    'gulp',
    'babel-cli',
    'webdriverio',
    'updtr',
    'junit-viewer',
    'html-to-react-components',
  ],
  osx: [
    '1Password',
    'Buffer',
    'Brightness menu bar',
    'Color Picker',
    'Copied',
    'Cloud Outliner 2 Pro',
    'Degrees',
    'DrCleaner',
    'Due',
    'HazeOver',
    'Irvue',
    'iTranslate',
    'Media Player',
    'MenuTab for Facebook',
    'Microsoft OneNote',
    'Momentum',
    'Magnet',
    'NoteAway',
    'Noizio',
    'Numbers',
    'OmniFocus',
    'Paste',
    'Pomodoro Time',
    'Pull - Menu Timer',
    'Raindrop.io',
    'Reeder',
    'Simplify',
    'Slack',
    'SmartConverter',
    'Twitter',
    'Unclutter',
  ]
};
